### ISSUE TITLE
Adding expected failure tests

### ISSUE THREAD

**Issue by chf3kl9** [Tournamanager/api#86]
What part needs to be improved:
Add tests for failures for addUserToTeam (#60).

Why does the part need to be improved:
In #83 we added the functionality to add users to a team, but the tests that are written are only testing sucesses e.g. adding user 1 to team 1 will result in team 1 having user 1 as team member.
What we forgot to test is testing exceptions e.g. adding user 1 to team 1 again will result in an exception.
tldr. besides testing whether the code succeeds where we expect it to succeed, we should also test whether the code fails where we expect it to fail.

When is the part improved (DoD):
When we are testing the expected failures of addUserToTeam (#60)
